### PR TITLE
[Artemis] Changes to verify ARTEMIS-2852

### PR DESCRIPTION
**Pull request by ghostbuster91**

### CODE REVIEW

**Review by franz1981 — APPROVED** [review]
LGTM

File: ansible/roles/artemis/tasks/main.yml [hunk 1]
```diff
@@ -237,7 +237,14 @@
   replace:
     path: /opt/{{ artemis_name }}-broker/etc/artemis.profile
     regexp: 'Xmx2G'
-    replace: 'Xmx48G'
+    replace: 'Xmx16G'
```
Comment (michaelandrepearce) [inline ansible/roles/artemis/tasks/main.yml:240]: ??

File: ansible/roles/artemis/tasks/main.yml [hunk 2]
```diff
@@ -237,7 +237,14 @@
...
+    backup: yes
+
+- name: Drop UseStringDeduplication
+  replace:
+    path: /opt/{{ artemis_name }}-broker/etc/artemis.profile
+    regexp: '-XX:\+UseStringDeduplication'
```
Comment (michaelandrepearce) [inline ansible/roles/artemis/tasks/main.yml:246]: Again really not needed

File: ansible/roles/artemis/tasks/main.yml (hunk 1 above)
Comment (michaelandrepearce) [reply ansible/roles/artemis/tasks/main.yml:240]: This change is not needed.

Comment (franz1981) [reply ansible/roles/artemis/tasks/main.yml:240]: This is preferred to have some apple 2 apple comparisons...G1 isn't that good with huge heaps as it has been sold 🤣

**Pull request closed without merging**